examples/polypharmacy_gripnet/utils.py

```diff
@@ -1,93 +1,84 @@
-import pickle
+import imp
```

Comment:
This is the only example with utils.py. That's why it should be refactored. No any other example has this so this should be refactored into elsewhere to be consistent.